sources/cc-0812-pres-meet-cc/pres-bedework.adoc

```diff
@@ -0,0 +1,354 @@
+= CalConnect and Open Source
+:docnumber: 0812
+:copyright-year: 2008
+:language: en
+:doctype: administrative
+:edition: 1
+:status: published
+:revdate: 2008-11-07
+:published-date: 2008-11-07
+:technical-committee: CALCONNECT
+:mn-document-class: cc
+:mn-output-extensions: xml,html,pdf,rxl
+:local-cache-only:
+:fullname: Gary Schwartz
+:affiliation: Communications & Middleware Technologies, Rensselaer Polytechnic Institute
+:contributor-position: Director
+:imagesdir: images
+
+.Foreword
+
+This document incorporates by reference the CalConnect Intellectual Property Rights, Appropriate Usage, Trademarks
+and Disclaimer of Warranty for External (Public) Documents as located at
+
+http://www.calconnect.org/documents/disclaimerpublic.pdf.
+
+[IMPORTANT]
+====
+All documents included in the zip file for Meet CalConnect 2008 are considered part of CD 0812 and the public disclaimer included by reference above applies to all contents of the file.
+====
+
+== Please Allow Me to Introduce Myself
+
+Gary Schwartz
+
+** Director, Communications & Middleware Technologies, Rensselaer Polytechnic Institute
```

Comment:
Should be one level list.

Reply:
This is resolved in https://github.com/CalConnect/cc-admin-documents/pull/17/commits/8c87f5fd8ff1ed9c5a5b45f0874728c4f6c4d91a.